libs/SalesforceNetwork/SalesforceNetwork/SalesforceNetwork/Classes/Network/Action/CSFSalesforceAction.m

```diff
@@ -273,4 +273,9 @@ - (void)userAccountManagerDidChangeCurrentUser:(NSNotification*)notification {
     }
 }
 
+- (NSURLRequest*)createURLRequest:(NSError**)error {
+    self.baseURL = self.enqueuedNetwork.account.credentials.apiUrl;
```

Comment:
Setting ```baseURL``` in the subclass (this one's for Salesforce specific actions). Ideally, we would want this in the ```init``` method, but sadly that's not possible, because setting the ```baseURL``` requires ```enqueuedNetwork``` to be setup first, which only happens on ```start```, which in turn only happens when the action is enqueued on the network object (can't be done before the action's ```init``` happens, and it depends on the caller). Long story short - this is where it was being set before, except that it was being set in ```CSFAction```, and now it's set in the subclass.